gfauto/gfauto/amber_converter.py

```diff
@@ -261,7 +261,13 @@ def amberscript_uniform_buffer_bind(uniform_json: str, prefix: str) -> str:
     for name, entry in uniforms.items():
         if name.startswith("$"):
             continue
-        result += f"  BIND BUFFER {prefix}_{name} AS uniform DESCRIPTOR_SET 0 BINDING {entry['binding']}\n"
+        if "binding" in entry.keys():
```

Comment:
Inside this if, perhaps assert that "push_constant" is not a key - to enforce that they are mutually exclusive.

Reply:
Done